football_dictionaries/assignment_3.py

```diff
@@ -1,2 +1,17 @@
+from .assignment_1 import players_as_dictionaries
+
 def players_by_country_and_position(squads_list):
-    pass
+    player_country = {}
+    
+    for player in players_as_dictionaries(squads_list):
+        if player["country"] not in player_country:
+            player_country.update({player["country"]:{}})
+            player_country[player["country"]].update({player['position']:[]})
+            player_country[player["country"]][player['position']].append(player)
+        elif player["position"] not in player_country[player["country"]]:
+            player_country[player["country"]].update({player['position']:[]})
+            player_country[player["country"]][player['position']].append(player)
+        else:
+            player_country[player["country"]][player['position']].append(player)
+
+    return player_country
```

Comment:
I like the solution you used here. I used separate if statements originally instead of if/elif/else but seems this will limit the amount of lines each item in the for loop will go through.